src/components/grid/hovered_cell_store.ts

```diff
@@ -0,0 +1,24 @@
+import { SpreadsheetStore } from "../../store_engine/spreadsheet_store";
+import { Command, Position } from "../../types";
+
+export class HoveredCellStore extends SpreadsheetStore {
+  col: number | undefined;
+  row: number | undefined;
+
+  handle(cmd: Command) {
+    switch (cmd.type) {
+      case "ACTIVATE_SHEET":
+        this.clear();
+    }
+  }
+
+  hover(position: Position) {
+    this.col = position?.col;
```

Comment:
Is the "?" really needed ? Same question for the line under

Reply:
indeed, it's a relicate of the dev. tahnks for spotting it :)